filter_linux.go

```diff
@@ -130,6 +132,22 @@ func (filter *Flower) encode(parent *nl.RtAttr) error {
 		parent.AddRtAttr(nl.TCA_FLOWER_KEY_ENC_KEY_ID, htonl(filter.EncKeyId))
 	}
 
+	if filter.SkipHw {
```

Comment:
should they both be added to the same TCA_FLOWER_FLAGS attribute ?

Reply:
What do you mean? We can't add both TCA_CLS_FLAGS_SKIP_HW and TCA_CLS_FLAGS_SKIP_SW at the same time

Reply:
Yes i realize its invalid config (having both set).

I think, in general, if there are multiple fields on the same attribute they should be set together, if its invalid then netlink(kernel) will fail it (i hope).

does it fail when both are set on the same attribute ?

Reply:
`~# tc filter add dev enp3s0f0np0 protocol ip parent ffff: flower ip_proto 3 skip_hw skip_sw action drop
RTNETLINK answers: Invalid argument
We have an error talking to the kernel`

Reply:
ack, thx for clarifying. i see you also changed flags it to be on the same attr thx !